molo/core/models.py

```diff
@@ -107,14 +107,6 @@ class ArticleOrderingChoices(enum.Enum):
     PK = 4
     PK_DESC = 5
 
-    labels = {
-        CMS_DEFAULT_SORTING: 'CMS Default Sorting',
-        FIRST_PUBLISHED_AT: 'First Published At',
-        FIRST_PUBLISHED_AT_DESC: 'First Published At Desc',
-        PK: 'Primary Key',
-        PK_DESC: 'Primary Key Desc',
-    }
-
```

Comment:
Don't we use these labels for the site settings?

Reply:
added in django built in enum support

Reply:
changed on top